### DISCUSSION TITLE
Change The Line Style

### DISCUSSION THREAD

**Discussion by QuangLanh** [clientIO/joint#2417] [Q&A]
### Introduction

How can I indicate when two paths intersect with each other so that users can distinguish whether the two paths are connected or not?
I want to achieve what is shown in the example below. Is it possible?

### Steps to reproduce

![image](https://github.com/clientIO/joint/assets/91791283/a7b14445-630d-4a79-a6b6-717ce3e74237)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
You can use [jumpover](https://resources.jointjs.com/docs/jointjs/v3.7/joint.html#connectors.jumpover) connector or make the link outline color to match the color of the paper's background as shown [here](https://www.jointjs.com/demos/flowchart) (note that link is usually created with 2 SVGPaths - one is representing the actual line and the other one under is for pointer-events but can be made visible).

**Reply by QuangLanh** [reply]
I have successfully used the 'connector.jumpover' as per your instructions. However, there is still an issue: how can users create connections from one link to another, like the example above? Please guide me. Thank you.

**Reply by kumilingus** [reply]
You can use [linkTools.Connect](https://resources.jointjs.com/docs/jointjs/v3.7/joint.html#linkTools.Connect) or [linkTools.HoverConnect](https://resources.jointjs.com/docs/jointjs/v3.7/joint.html#linkTools.HoverConnect).

Here is an example: https://www.jointjs.com/demos/sequence-diagram
